snactor/registry/actors.py

```diff
@@ -19,6 +19,13 @@ def must_get_actor(actor):
     return _instantiate_actor(_REGISTERED_ACTORS[actor])
 
 
+def actor_names_by_tags(tags, names=()):
+    if names == ():
+        return [name for name, a  in _REGISTERED_ACTORS.items() if a[0].tags.intersection(tags)]
```

Comment:
intersection isn't checking for subsets so if you want that then do check for subsets  
```python
return [name for name, a  in _REGISTERED_ACTORS.items() if a[0].tags.issubset(tags)]
```